### ISSUE TITLE
Minor error in example used in the readme file.

### ISSUE THREAD

**Issue by jaufgang** [tc39/proposal-javascript-standard-library#21]
in the  [Generic Functions](https://github.com/tc39/proposal-javascript-standard-library/blob/master/README.md#generic-functions) section of the `README.md` file, you have 

```js
const mp = new Map([["e", 8], ["n", 9], ["z", 0], ["o", 1], ["t", 2]);
```
which has 5 elements, the last one being `["t",2]`, however the examples for the `len` and `map` functions reflect a value for `mp` that only has 4 elements and is missing the final element `t` .  

I think this is just a minor typo, but it jumped out at me and should be fixed.

**Comment by littledan** [comment]
Thanks for pointing this out. PRs welcome.

**Comment by jaufgang** [comment]
resolved by #5
